hack/generate-manifest.sh

```diff
@@ -46,6 +46,7 @@ Generate a YAML manifest for Antrea using Helm and print it to stdout.
         --verbose-log                 Generate a manifest with increased log-level (level 4) for Antrea agent and controller.
                                       This option will work only in 'dev' mode.
         --help, -h                    Print this message and exit.
+        --custom-img-tag              Custom tag for Antrea images.
```

Comment:
this duplicates the functionality provided by the `IMG_TAG` environment variable, and should not be added

Reply:
has not been removed yet

Reply:
no change here, the flag is still present?

Reply:
no the flag is removed, I have updated the script as well, thanks for pointing this out.